lang/en.json

```diff
@@ -50,7 +50,6 @@
   "ffg-star-wars-enhancements.attack-animation.custom.notification-success": "Set custom animation data!",
   "ffg-star-wars-enhancements.attack-animation.custom.notification-unset": "Custom animation was removed",
   "ffg-star-wars-enhancements.attack-animation.custom.global": "(global value)",
-  "ffg-star-wars-enhancements.controls.opening-crawl.title": "Opening Crawl",
```

Comment:
@ercete why was this line removed?

Reply:
@ercete looking through `opening_crawl.js`, this appears to be in use. Can you make a commit adding this line back (in both English and French) and then I can accept this PR.